promql/parse_test.go

```diff
@@ -1333,12 +1332,6 @@ var testStatement = []struct {
 	}, {
 		input: `foo{a!~"b"} = bar`,
 		fail:  true,
-	}, {
-		input: `ALERT SomeName IF time() LABELS {} 
-			SUMMARY "Global request rate low"
-			DESCRIPTION "The global request rate is low"
-		`,
-		fail: true,
```

Comment:
How about testing both the old and new versions here? Either just duplicate the expected result, or check the parsing of the new style, then check that the result of the old and new style is the same?